src/ray/pubsub/publisher.cc

```diff
@@ -22,19 +22,17 @@ namespace pubsub {
 
 namespace pub_internal {
 
-bool EntityState::Publish(std::shared_ptr<rpc::PubMessage> msg) {
+bool EntityState::Publish(std::shared_ptr<rpc::PubMessage> msg, uint64_t msg_size) {
```

Comment:
size_t

```

virtual size_t | ByteSizeLong() constComputes the serialized size of the message. more...
-- | --


virtual size_t	
ByteSizeLong() const
Computes the serialized size of the message. [more...](https://protobuf.dev/reference/cpp/api-docs/google.protobuf.message/#Message.ByteSizeLong.details)
```

Reply:
I used `uint64_t` since it's used elsewhere...